src/app/client/src/app/modules/shared/components/page-section/page-section.component.html

```diff
@@ -17,15 +17,15 @@ <h4 class="ui header m-0 mr-5 pageSection-heading">{{section.name}}</h4>
                 class="carousel" 
                 #slickModal="slick-modal" 
                 [config]="slideConfig">
-                <div in-view-item [id]="i" [data]="content" ngxSlickItem *ngFor="let content of section.contents;let i = index" class="slide">    
+                <div in-view-item [id]="i" [data]="content" ngxSlickItem *ngFor="let content of section.contents;let i = index; let last = last" class="slide">    
                     <app-card 
                         appTelemetryInteract 
                         [telemetryInteractEdata]="cardIntractEdata" 
                         [telemetryInteractObject]="{id:content.metaData.identifier || content.metaData.courseId,type:content.metaData.contentType || 'Course',ver:'1.0'}"
                         (clickEvent)="playContent($event)"
                         [data]="content" 
                         [customClass]="'mr-20'"
-                        class="pr-20 d-block">
+                        class="pr-20 d-block" [class.last-card]="last">
```

Comment:
@prajwal88 Instead of adding class ```last-card``` use class ```pr-2```.  and create a reusable class ```pr-2``` for both variations - ltr and rtl.